<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingFilterHorizontalBand</class>
 <widget class="QDialog" name="SettingFilterHorizontalBand">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxGain">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>J:/検査画面icon/flash_swf_64.PNG</normaloff>J:/検査画面icon/flash_swf_64.PNG</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>J:/検査画面icon/flash_64.PNG</normaloff>J:/検査画面icon/flash_64.PNG</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>113</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gain</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Center X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Band width</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gradation width</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxCenterX">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxBandWidth">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxGradationWidth">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0 : not change</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbLayer2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbLayer0">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer 0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbLayer1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>283</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pixel Gamma</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxGamma">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
